### ISSUE TITLE
Attribution/Credits in "Satisfying Real World Use Cases"

### ISSUE THREAD

**Issue by ChristopherA** [WebOfTrustInfo/rebooting-the-web-of-trust#53]
/cc https://github.com/WebOfTrustInfo/rebooting-the-web-of-trust/blob/985eb6010de59303197851173edb82c535d34def/draft-documents/satisfying-real-world-use-cases.md
/re @du5t 

How does your team wish to handle attribution? Just "The Participants of the December 2015 #RebootingWebOfTrust Workshop"? A single list of everyone at the workshop? A more narrow list? A list with roles?

Which choice will make the final white paper, edited by @shannona and with graphics from Sonia, more effective to a broader audience?

**Comment by du5t** [comment]
Good question. Just for the record, here is a list of people who contributed to the document itself and their previous attribution preference (yes/no) if they gave one:
- @du5t - yes
- @identitywoman - yes
- @kiaraRobles - no
- @talltree 
- @aquabu 
- John Edge

**Comment by du5t** [comment]
Perhaps a single "authorship" page at the end with a short list like the above with brief information like roles, and a longer list of the workshop attendees and workshop roles/titles? That would arguably represent the most information in context.

**Comment by taoeffect** [comment]
If it helps, in our paper we have author attributions per section, and I'm also thinking we might have an "Authors" or "Acknowledgements" part at the top with all names listed (perhaps including reviewers). Just some ideas we're considering that might be worth thinking about.

**Comment by jbenet** [comment]
> If it helps, in our paper we have author attributions per section, and
> I'm also thinking we might have an "Authors" or "Acknowledgements" part at
> the top with all names listed (perhaps including reviewers). Just some
> ideas we're considering that might be worth thinking about.

yeah i think this is the right way. and acknowledgements to the "Rebooting
Web of Trust Group" can be a catch all for all the smaller contribs.

**Comment by talltree** [comment]
+1

**Comment by du5t** [comment]
For now I've added a byline, but I like where this conversation is going. Let me know when you settle on something and we can slide it in.
